ouroboros-consensus-cardano-tools/src/Cardano/Tools/DBAnalyser/Block/Cardano.hs

```diff
@@ -185,13 +192,14 @@ instance AdjustFilePaths CardanoConfig where
             byronGenesisPath    = f $ byronGenesisPath cc
           , shelleyGenesisPath  = f $ shelleyGenesisPath cc
           , alonzoGenesisPath   = f $ alonzoGenesisPath cc
+          , conwayGenesisPath   = f $ conwayGenesisPath cc
           }
 
 -- | Shelley transition arguments
 data STA :: Type -> Type where
   STA ::
-       -- so far, the context is either () or AlonzoGenesis
-       (SL.AlonzoGenesis -> Core.TranslationContext era)
+       -- so far, the context is either () or AlonzoGenesis or ConwayGenesis
+       ((SL.AlonzoGenesis, SL.ConwayGenesis StandardCrypto) -> Core.TranslationContext era)
```

Comment:
Why do we need to change this type?

Reply:
My reply to your next comment will probably help explain it. In the context of what I wrote there, I'd answer your question here with "because Alonzo and Conway are so far the only Shelley-based eras that add new genesis data; the Shelley era did too of course, but this code is about translating from a Shelley-based era to a Shelley-based era, and the actual Shelley era didn't translate from a Shelley-based era"